acceptance/tests/options/config_file/invalid_file.rb

```diff
@@ -0,0 +1,30 @@
+test_name "FACT-2923: providing an invalid config file prints a warning" do
```

Comment:
Do we need an acceptance test for this? We can just force garbage into `Hocon.load` and test how facter behaves there(like it calls the log.warn)

Reply:
Done